packages/wonder-blocks-cell/src/components/internal/cell-core.tsx

```diff
@@ -201,28 +178,52 @@ const CellCore = (props: CellCoreProps): React.ReactElement => {
                 hideDefaultFocusRing={true}
                 aria-label={ariaLabel ? ariaLabel : undefined}
                 target={target}
+                style={[
+                    styles.wrapper,
+                    styles.clickable,
+                    active && styles.active,
+                    disabled && styles.disabled,
+                ]}
+                aria-current={active ? "true" : undefined}
             >
-                {(eventState) => renderCell(eventState)}
+                {() => renderCell()}
             </Clickable>
         );
     }
 
     // No click event attached, so just render the cell as-is.
-    return renderCell();
+    return (
+        <View
+            style={[styles.wrapper, active && styles.active]}
+            aria-current={active ? "true" : undefined}
+        >
+            {renderCell()}
+        </View>
+    );
 };
 
 const styles = StyleSheet.create({
     wrapper: {
         background: Color.white,
         color: Color.offBlack,
-        minHeight: CellMeasurements.cellMinHeight,
+        // minHeight: CellMeasurements.cellMinHeight,
         textAlign: "left",
     },
 
     innerWrapper: {
+        // minHeight: CellMeasurements.cellMinHeight,
```

Comment:
Did you mean to remove this?

Reply:
ahh good call! I was testing some stuff, but forgot to revert those changes.